test/traversal/forEachDescriptor.js

```diff
@@ -124,8 +124,8 @@ describe('forEachDescriptor', () => {
         variations: './to/variations/**',
       };
 
-      const variationPathA = `${projectRoot}/${variationsRoot}/path/to/a`;
-      const variationPathB = `${projectRoot}/${variationsRoot}/path/to/b`;
+      const variationPathA = `${projectRoot}/${variationsRoot}path/to/a`;
```

Comment:
Previously, this was giving a `//` since variationsRoot contained a trailing slash.